Biasing/src/Biasing/EcalDarkBremFilter.cxx

```diff
@@ -130,11 +112,11 @@ bool EcalDarkBremFilter::inDesiredVolume(const G4Track* track) const {
 
 void EcalDarkBremFilter::AbortEvent(const std::string& reason) const {
   if (G4RunManager::GetRunManager()->GetVerboseLevel() > 1) {
```

Comment:
I think we should remove these Geant4 verbose level guards now that the messages are in the logging. I want to avoid having to do two things in order to see a message.

Reply:
I agree, done in done in https://github.com/LDMX-Software/ldmx-sw/pull/1767/commits/6b869b0b771b66170058a7f34f31c68e7f24fa95